kedro/framework/project/__init__.py

```diff
@@ -199,8 +200,11 @@ def __init__(self):
 
         # We suppress click here to hide tracebacks related to it conversely,
         # kedro is not suppressed to show its tracebacks for easier debugging.
-        # sys.executable is used to get the kedro executable path to hide the top level traceback.
-        rich.traceback.install(suppress=[click, str(Path(sys.executable).parent)])
+        # sys.executable is used to get the kedro executable path to hide the top level traceback.   
+        # Rich traceback handling does not work on databricks. 
```

Comment:
```suggestion
        # Rich traceback handling does not work on databricks. 
        # Hopefully this will be fixed on their side at some point, but until then we disable it.
```